RELEASE/scripts/sl_ascend/sl_koe.ash

```diff
@@ -19,3 +19,65 @@ boolean koe_initializeSettings()
 	}
 	return false;
 }
+
+boolean LX_koeInvaderHandler()
+{
+	if(!in_koe())
+	{
+		return false;
+	}
+
+	buffMaintain($effect[Astral Shell], 10, 1, 1);
+	buffMaintain($effect[Elemental Saucesphere], 10, 1, 1);
+	buffMaintain($effect[Hide of Sobek], 10, 1, 1);
+	buffMaintain($effect[Spectral Awareness], 10, 1, 1);

```

Comment:
I don't think we're going to have Hide of Sobek or Spectral Awareness in KoE :P

Reply:
😆